web/src/lightbox.ts

```diff
@@ -634,7 +637,9 @@ export function initialize(): void {
         e.stopPropagation();
 
         const $video = $(e.currentTarget).find<HTMLMediaElement>("video");
+        $video.addClass("media-to-select-in-lightbox-list");
         open_video($video);
+        $video.removeClass("media-to-select-in-lightbox-list");
```

Comment:
This seems fragile, in that if an exception is thrown in `open_video`, we'll fail to clean up the class. This can probably be addressed by just doing `$(".media-to-select-in-lightbox-list").removeClass("media-to-select-in-lightbox-list") before the addClass (probably making that a small helper function since all 3 code paths would use the same pattern).